pimd/pim_igmp.c

```diff
@@ -1113,8 +1113,10 @@ struct igmp_group *igmp_add_group_by_addr(struct igmp_sock *igmp,
 	}
 
 	if (pim_is_group_224_0_0_0_24(group_addr)) {
-		zlog_warn("%s: Group specified is part of 224.0.0.0/24",
-			  __PRETTY_FUNCTION__);
+		if (PIM_DEBUG_IGMP_TRACE)
+			zlog_debug(
+				"%s: Group specified %s is part of 224.0.0.0/24",
+				inet_ntoa(group_addr), __PRETTY_FUNCTION__);
```

Comment:
shouldn’t be pretty as first argument?

Reply:
yes, and it should probably be just `__func__`, since pretty is the same

Reply:
I'm going to continue to use __PRETTY_FUNCTION__ in pim since that is what the code base has for consistency sake.

Reply:
sure